### DISCUSSION TITLE
P2.1 of UIA vs Scarlet dev update

### DISCUSSION THREAD

**Discussion by Ministry-of-CBT** [Nrmot-GithubRepo/Project-CBT#14] [Project CBT: Latest Dev Releases]
changes:

tied all assets into one modpack after a refresh, thats all

also a few quests got gregified

also aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa there's only client in the dev release no server files aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa its fine 

tomorrow i will probably finish this part of the update, since i got home a few hours ago.

**Full Changelog**: https://github.com/Nrmot-GithubRepo/Project-CBT/compare/v0.0.0-dev7...v0.0.0-dev8

<hr /><em>This discussion was created from the release <a href='https://github.com/Nrmot-GithubRepo/Project-CBT/releases/tag/v0.0.0-dev8'>P2.1 of UIA vs Scarlet dev update</a>.</em>

**Comment by Higuys153** [comment]
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa ok aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa yes it is fine
